src/common/components/placeSelector/__tests__/PlaceSelector.test.tsx

```diff
@@ -1,20 +1,23 @@
-import range from 'lodash/range';
 import React from 'react';
 
+import { cache } from '../../../../domain/app/apollo/apolloClient';
 import { PlaceDocument, PlacesDocument } from '../../../../generated/graphql';
 import { fakePlace, fakePlaces } from '../../../../utils/mockDataUtils';
 import {
+  configure,
   render,
   screen,
   userEvent,
   waitFor,
 } from '../../../../utils/testUtils';
 import PlaceSelector, { PlaceSelectorProps } from '../PlaceSelector';
 
+configure({ defaultHidden: true });
+afterEach(() => cache.reset());
```

Comment:
Good idea. It doesn't take much effort to add cache.reset() to each test file. However, in the future it might cause some time consuming test debugging when trying to find out why some tests are failing.

Reply:
And done